.github/workflows/push-build-downstream.yaml

```diff
@@ -0,0 +1,81 @@
+name: Push Image Build Downstream
+on:
+  push:
+    branches:
+      - master
+      - release-*
+    tags:
+      - v*
+  pull_request:
+    branches:
+      - master
```

Comment:
Basically, this can be removed

Reply:
right, i added this just for the testing